scripts/orthomcl-setup-database.pl

```diff
@@ -93,3 +130,8 @@ sub parse_config
 
 	return \%parameters;
 }
+
+END
+{
+    $dbh->disconnect();
```

Comment:
If you run the script without any parameters I get the following:

```
$ perl orthomcl-setup-database.pl 
missing database information
Usage: orthomcl-setup-database.pl --user [user] --password [password] --host [host] --database [database] --outfile [outfile]
Checks for a database connection and generates an orthomcl config file with the given parameters
Can't call method "disconnect" on an undefined value at orthomcl-setup-database.pl line 136.
END failed--call queue aborted at orthomcl-setup-database.pl line 39.
```

Could you fix this up?  I assume it's because END gets called if you run `die`, and the program will sometimes die before a database is connected to.